src/Map/doc/index.rst

```diff
@@ -0,0 +1,211 @@
+Symfony UX Map
+==============
+
+**EXPERIMENTAL** This component is currently experimental and is likely
+to change, or even change drastically.
+
+Symfony UX Map is a Symfony bundle integrating interactive Maps in
+Symfony applications. It is part of `the Symfony UX initiative`_.
+
+Installation
+------------
+
+.. caution::
+
+    Before you start, make sure you have `StimulusBundle configured in your app`_.
+
+Install the bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-map
+
```

Comment:
If you're using AssetMapper (recommanded solution, configured if you installed your project with ``symfony new --webapp```), you're ready to go!     [...]

Then webpack encore ?